### DISCUSSION TITLE
Cant do any foundry function

### DISCUSSION THREAD

**Discussion by pakhomovvv** [foundry-rs/foundry#8909] [Q&A] [answered]
Well, I tried to install docker, but it looks like i have to have the 12+ version of mac, and i have 11, so i tried installing it on the command line, using HOMEBREW (everytime i install homebrew smth bad happens).
So when i was installing it, i cancelled the instalation bc i knew this wouldnt work and i was waiting for a long time.
So i asked chatgpt to help me deleting it completely, and it looks like i deleted smth from foundry, because i cant do any forge function nor anvil commnd. Should i do everything from zero, or is there a solution for this problem?

**Comment by onbjerg** [comment, marked as answer]
You should be able to just run `foundryup` and it should work again. If not, you uninstalled something that might be more important to your system, but it's not possible to say so without you posting the error you get when running a forge command
